plugins/qtAccessionPlugin/js/validation.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {
+  function showOrRemoveIdentifierError(allowable, message) {
+    // Remove any currently displayed identifer errors
+    $('.identifier-error').remove();
+
+    // Show error if identifier not available
+    if (!allowable)
+    {
+      $messageHtml = '<div class="identifier-error alert alert-danger">' + message + '</div>';
+      $('#identifier').after($messageHtml);
+    }
+  }
+
+  $(function() {
+    var formValid = false;
+
+    // If identifier is already populated, then it has been validated by
+    // server-side logic
+    if ($('#identifier').val()) {
+      formValid = true;
+    }
+
+    // Disable form submission if form isn't valid or there's no identifier
+    $('#editForm').submit(function(e){
+      if (!formValid || !$('#identifier').val())
+      {
+        e.preventDefault();
+      }
+    });
+
+    // Activate identifier pre-validation
+    $('#identifier').change(function() {
+      var url = $('#identifierAvailableCheckUrl').val();
+
+      $.ajax({
+        url: url + '&identifier=' + $('#identifier').val(),
+
+        beforeSend: function() {
+          $('#spinner').show();
+        },
+
+        success: function(data) {
+          $('#identifier-check-server-error').hide();
+          formValid = data['allowable'];
+          showOrRemoveIdentifierError(data['allowable'], data['message']);
+        },
+
+        complete: function () {
+          $('#spinner').hide();
+        },
+
+        error: function (jqXHR, textStatus, thrownError) {
+          $('#identifier-check-server-error').show();
+        }
+      });
+    });
+  });
+})(window.jQuery);  
```

Comment:
Can you try to instantiate jQuery objects only once? e.g. `$('#identifier')` is used three times in the same function and perhaps some of the elements could be reached faster just by using `.find()` instead?